packages/expo/tsconfig.json

```diff
@@ -1,6 +1,26 @@
 {
-  "extends": "./tsconfig.build.json",
   "compilerOptions": {
+    "outDir": "dist",
```

Comment:
We should extend this from a root/base tsconfig.json

But that's for another PR